com.note_slayer.app.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.note_slayer.app
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
```

Comment:
does it need sound access?

Reply:
Thank you for pointing that out, i don't need I'll remove that